### PR TITLE
Exclude org.checkerframework:checker-qual from runtime dependencies

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: bom/runtime/pom.xml [hunk 1]
```diff
@@ -976,6 +976,12 @@
                 <groupId>com.github.ben-manes.caffeine</groupId>
                 <artifactId>caffeine</artifactId>
                 <version>${caffeine.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.checkerframework</groupId>
+                        <artifactId>checker-qual</artifactId>
+                    </exclusion>
+                </exclusions>
```
Comment (gsmet) [inline bom/runtime/pom.xml]: If we know it's an issue, can we add it to the banned dependencies of the enforcer in the build-parent?

**Review by gsmet — CHANGES_REQUESTED** [review]

File: bom/runtime/pom.xml (hunk 1 above)
Comment (michalszynkiewicz) [reply bom/runtime/pom.xml]: It also goes in through maven-embedder but then it is only used in deployment. 
I'd need to exclude it there too to ban it.
I'll give it a try.

**Comment by gsmet** [comment]
Ok if it’s not obvious, we can live with it. We have better things to do :)

**Review by gsmet — APPROVED** [review]
Feel free to merge when you think it’s ready.

**Comment by michalszynkiewicz** [comment]
@gsmet thanks for mentioning banning. It turned out there was a few places I missed when simply looking at the dependency tree.

**Pull request merged**